src/cryptography/hazmat/primitives/serialization/base.py

```diff
@@ -4,38 +4,47 @@
 
 
 import abc
+import typing
 from enum import Enum
 
 from cryptography import utils
 from cryptography.hazmat.backends import _get_backend
+from cryptography.hazmat.primitives.asymmetric import dh
 
+if typing.TYPE_CHECKING:  # pragma: no cover
```

Comment:
I don't like this, this is making our code messier and more confusing for the type checker. Can we put these constants in some module that avoids the circular issue?